components/scream/src/mct_coupling/atm_comp_mct.F90

```diff
@@ -322,8 +319,10 @@ subroutine atm_domain_mct( lsize, gsMap_atm, dom_atm )
     call scream_get_cols_latlon(c_loc(data1),c_loc(data2))
     call mct_gGrid_importRAttr(dom_atm,"lat",data1,lsize) 
     call mct_gGrid_importRAttr(dom_atm,"lon",data2,lsize) 
-    data1(:) = data1(:) * rad2deg
-    data2(:) = data2(:) * rad2deg
+
+    call mpi_comm_rank(mpicom_atm, rank, ierr)
```

Comment:
This seems unrelated to the fix, but I'm guessing is just another chunk of code we need so might as well put it here?

Reply:
Ah, I put it in while debugging. Let me prune it.